### ISSUE TITLE
[thumbnail] Upload a new avatar does not update

### ISSUE THREAD

**Issue by pyphilia** [graasp/graasp#685]
If you change your avatar, everything goes as expected but the image does not update in builder. Cache problem?

**Comment by spaenleh** [comment]
@pyphilia Yes this is entirely an issue in our setup for the member avatar query. We have a cacheTime set to 15min. This means that the thumbnail is not updated even when we invalidate the query. We should probably remove that cache time as it does not really help us here. To avoid re-fetching too often, we could instead use a staleTime that is longer: 15min for example. This would indicate that the resource is fresh for longer, but will still allow us to invalidate it.
